packages/taquito-rpc/src/types.ts

```diff
@@ -677,7 +687,9 @@ export interface BlockMetadata {
   max_operation_list_length: MaxOperationListLength[];
   baker: string;
   level: Level;
+  level_info: LevelInfo;
```

Comment:
I think level, level_info, voting_period_kind, and voting_period_info should be optional to stay compatible.


Deprecated: metadata.level, metadata.voting_period_kind
New properties: metadata.level_info, metadata.voting_period_info

Reply:
Thank you for the feedback, I have made these properties optional.